### ISSUE TITLE
Licensed doesn't handle a nil root for relative path expansion

### ISSUE THREAD

**Issue by jonabc** [github/licensed#95]
Licensed needs an absolute path to use as a root for all relative path expansions from configuration files and CLI output.

Currently, licensed assumes that it's run from a git repository and uses the repository root as the absolute path to expand all relative paths.  If licensed isn't run from a git repository though, everything will blow up.

In https://github.com/github/licensed/pull/94, a solution is proposed that allows a custom absolute path `root` value to be set but still doesn't handle the case when the root is unset and a git repository root is unavailable.

A `nil` root could be handled by 
1. setting a default root value if all other options aren't available
  - `pwd` maybe?
2. throwing an explicit error if a root value isn't available in a configuration
3. ??

**Comment by mlinksva** [comment]
I'd probably _slightly_ prefer `pwd` so as to minimize required configuration when sometimes you just want to trying running the program...but I don't feel strongly at all.

**Comment by jonabc** [comment]
> so as to minimize required configuration when sometimes you just want to trying running the program

@mlinksva that makes sense

if the configuration is being parsed from a file then maybe the "right" default root is the configuration file's directory.

if the configuration is being created directly using `Configuration.new`, then we can use `pwd`?

**Comment by mlinksva** [comment]
Makes sense to me.

**Comment by jonabc** [comment]
closing this for now.  I updated https://github.com/github/licensed/pull/94 to default to `pwd` when not in a git repository.  I decided against using the configuration file's directory when loading the configuration from a file because it's one more rule thats likely to cause additional confusion depending on how licensed is run.

I'm _slightly_ uneasy about using `pwd` because it brings an edge case where `pwd` is different at the time of `Configuration.new` from when dependencies are enumerated.  In practice though I don't think that's going to be a problem very often.
